packages/ni-redirects/src/config.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+	"blobStoreUrl": "https://pinsstdocsboprodukw001.blob.core.windows.net/published-documents",
```

Comment:
Assuming we're deploying these rules via Terraform, is it possible to use plain text bindings and pass in the variables per-environment here: https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/worker_script#plain_text_binding